views/default/js/elgg/admin.js

```diff
@@ -0,0 +1,186 @@
+		var pluginGuid = ui.item.attr('id');
+		pluginGuid = pluginGuid.replace('elgg-object-', '');
+
+		elgg.action('admin/plugins/set_priority', {
+			data: {
+				plugin_guid: pluginGuid,
+				// we start at priority 1
+				priority: ui.item.index() + 1
+			},
+			success: function() {
+				// update plugins with priority dependences
+				var priorityDep = new RegExp(elgg.echo('ElggPlugin:Dependencies:Priority'));
+				ui.item.siblings().andSelf().each(function() {
+					if (priorityDep.test($(this).find('.elgg-dependency-requires').text())) {
+						updatePluginView($(this));
+					}
+				});
+			}
+		});
+	};
+
+	/**
+	 * Update the plugin view.
+	 *
+	 * @param {Object} pluginView Plugin view element to update
+	 * @return void
+	 */
+	updatePluginView = function(pluginView) {
+		// get guid from id like elgg-object-<guid>
+		var pluginGuid = pluginView.attr('id');
+		pluginGuid = pluginGuid.replace('elgg-object-', '');
+
+		elgg.get({
+			url: elgg.config.wwwroot + "ajax/view/object/plugin/full",
+			dataType: "html",
+			cache: false,
+			data: {
+				guid: pluginGuid,
+				display_reordering: true
+			},
+			success: function(htmlData) {
+				if (htmlData.length > 0) {
+					pluginView.html(htmlData);
+				}
+			}
+		});
+	};
+
+	/**
+	 * In-line editing for custom profile fields
+	 *
+	 * @param string   value    The new value
+	 * @param {Object} settings The settings used for editable
+	 * @return void
+	 */
+	editProfileField = function(value, settings) {
+		var id = $(this).attr('id');
+		id = id.replace('elgg-profile-field-', '');
+
+		var data = {
+			id:    id,
+			label: value
+		};
+
+		elgg.action('profile/fields/edit', data);
+		return value;
+	};
+
+	/**
+	 * Save the plugin profile order after a move event.
+	 *
+	 * @param {Object} e  Event object.
+	 * @param {Object} ui jQueryUI object
+	 * @return void
+	 */
+	moveProfileField = function(e, ui) {
+		var orderArr = $('#elgg-profile-fields').sortable('toArray');
+		var orderStr = orderArr.join(',');
+
+		elgg.action('profile/fields/reorder', {
+			fieldorder: orderStr
+		});
+	};
+
+	/**
+	 * Fires the ajax action to delete the admin notice then hides the notice.
+	 *
+	 * @return void
+	 */
+	deleteNotice = function(e) {
+		e.preventDefault();
+		var $container = $(this).closest('p');
+
+		elgg.action($(this).attr('href'), {
+			success: function(json) {
+				$container.slideUp('medium');
+			}
+		});
+	};
+
+	/**
+	 * Toggles the display of the compression settings for simplecache
+	 *
+	 * @return void
+	 */
+	simplecacheToggle = function() {
+		// when the checkbox is disabled, do not toggle the compression checkboxes
+		if (!$(this).hasClass('elgg-state-disabled')) {
+			var names = ['simplecache_minify_js', 'simplecache_minify_css'];
+			for (var i = 0; i < names.length; i++) {
+				var $input = $('input[type!=hidden][name="' + names[i] + '"]');
+				if ($input.length) {
+					$input.parent().toggleClass('elgg-state-disabled');
+				}
+			}
+		}
+	};
+
```

Comment:
Yeah, I though I was already doing that, but it seems was returning it instead. Need to sleep more.